sql/moz-fx-data-shared-prod/udf/hr_cost_center_to_steering_committee/udf.sql

```diff
@@ -2,10 +2,16 @@
 This is a stub implementation for use with tests in this repo
 Real implementation is in private-bigquery-etl
 */
-CREATE OR REPLACE FUNCTION udf.hr_cost_center_to_steering_committee()
+CREATE OR REPLACE FUNCTION udf.hr_cost_center_to_steering_committee(
+  cost_center STRING,
+  cost_center_hierarchy_group STRING
+)
 RETURNS STRING AS (
-  'helloworld'
+  'steering_committee'
 );
 
 SELECT
-  mozfun.assert.equals(udf.hr_cost_center_to_steering_committee(), 'helloworld');
+  mozfun.assert.equals(
+    udf.hr_cost_center_to_steering_committee('cost_center', 'cost_center_heirarchy_group'),
+    'steering_committee'
+  );
```

Comment:
IMO having tests on these dummy UDFs doesn't serve a useful purpose, so if it were me I'd remove the tests.

Reply:
This is my first 'dummy' udf. I thought they were required. I can remove them.